Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/core/storage/TestCertificateContainer.kt

```diff
@@ -36,6 +36,7 @@ data class TestCertificateContainer(
 
     fun toTestCertificate(
         valueSet: TestCertificateValueSets?,
+        container: TestCertificateContainer,
```

Comment:
Do we need to pass the container? `toTestCertificate` is called inside `TestCertificateContainer`, we already have access to all of it's values.

Reply:
No all of them , I found that there are some values are needed from the containers such registredAt ,isPending, isUpdating